src/RacingGame.js

```diff
@@ -0,0 +1,44 @@
+import { Console } from '@woowacourse/mission-utils';
+
+class RacingGame {
+  constructor(cars, moveAttempts) {
+    this.cars = cars;
+    this.moveAttempts = moveAttempts;
+  }
+
+  play() {
+    Console.print('\n실행 결과');
+
+    for (let i = 0; i < this.moveAttempts; i += 1) {
+      this.cars.forEach(car => {
+        car.moveForward();
+      });
+      this.printCarMovement();
+
+      Console.print('\n');
+    }
+  }
+
+  printCarMovement() {
+    this.cars.forEach(car => {
+      Console.print(
+        `${car.getCarName()} : ${'-'.repeat(car.getMoveForwardCnt())}`,
+      );
+    });
+  }
```

Comment:
for문에서 cars 배열을 돌고, `printCarMovement`에서 cars 배열을 또 도니까
한 번만 돌도록 `moveForward` 뒤에 해당 자동차의 move를 출력하면 어떨까요?

```suggestion
    for (let i = 0; i < this.moveAttempts; i += 1) {
      this.cars.forEach(car => {
        car.moveForward();
        this.printCarMovement(car);
      });

      Console.print('\n');
    }
  }

  printCarMovement(car) {
      Console.print(`${car.getCarName()} : ${'-'.repeat(car.getMoveForwardCnt())}`);
  }
```

Reply:
저도 코드를 작성하면서 forEach() 메서드를 2번 쓰는게 맞나라는 고민이 있었는데 제시해주신 방법을 쓰면 해결되는 문제였네요...! 추천해주셔서 감사합니다.☺️